usr/bin/playtronos-session-select

```diff
@@ -0,0 +1,85 @@
+#! /bin/bash
+
+
+if [ $EUID -ne 0 ]; then
+	echo "$(basename $0) must be run as root"
+	exit 1
+fi
+
+
+SESSION_OVERRIDE="/etc/sddm.conf.d/60-playtron-session-override.conf"
+SESSION_LIST=('user' 'dev')
+SELECTED_SESSION="$1"
+
+if [ "$2" = "--no-switch" ]; then
+	NO_SWITCH=1
+
+	# a workaround to fix touch input in the dev session
+	systemctl restart inputplumber
+fi
+
+function dev() {
+	mkdir -p $(dirname $SESSION_OVERRIDE)
+
+	echo '
```

Comment:
A minor nit: this creates a newline at the beginning of the file. Do we want that?

Reply:
I went for readability of the code over the formatting of the file. It doesn't really matter.